src/service/github.ts

```diff
@@ -95,14 +95,15 @@ export async function createGitHubDeploymentAndJobSummary(
 		pagesArtifactFields.deployment_trigger
 	) {
 		const octokit = getOctokit(config.GITHUB_TOKEN);
+		const environment = config.DEPLOYMENT_NAME || pagesArtifactFields.environment;
```

Comment:
Hi!

First of all: thank you for this PR. I look forward to use it as I use the action to deploy just a documentation of my project and having "production" in deployments bugged me for a while

However: have you considered other name than `DEPLOYMENT_NAME`?

The line I'm pinning a thread to shows that the chosen name might be a bit misleading (deployment name/environment)

I'd suggest `DEPLOYMENT_ENVIRONMENT` as more precise. How about that?